dali/pipeline/operator/op_schema.h

```diff
@@ -387,6 +388,10 @@ class DLL_PUBLIC OpSchema {
     }
   }
 
+  DLL_PUBLIC inline const std::vector<std::vector<TensorLayout>> &SupportedLayouts() const {
```

Comment:
I think that it should be per-input, like GetInputDox and GetInputName.

Reply:
Done